src/utils/Error.js

```diff
@@ -0,0 +1,19 @@
+class HandleError {
+  static makeError(message) {
+    throw new Error(`[ERROR]: ${message}`);
+  }
+
+  static ATTEMPT_INPUT_ERROR = {
+    IsEmpty: '시도 횟수를 입력해주세요.',
```

Comment:
```suggestion
    IS_EMPTY: '시도 횟수를 입력해주세요.',
```
해당 메시지도 snake case를 사용하면 어떨까요?